sdcm/docker.py

```diff
@@ -157,7 +162,7 @@ def _update_image(self):
 
     def _create_container(self, node_name, is_seed=False, seed_ip=None):
         labels = f"--label 'test_id={cluster.Setup.test_id()}'"
-        cmd = f'run --cpus="1" --name {node_name} {labels} -d {self._node_img_tag}'
+        cmd = f'run --privileged --cpus="1" --name {node_name} {labels} -d {self._node_img_tag}'
```

Comment:
Why needs to run privileged ?

Reply:
I'm running it locally, and with hydra.
and didn't need the `systctl --system` nor the `--privileged`

I need more details why this is needed, cause I see nor reason it should.
in your log you called scylla directly without the correct commandline args.
please use `supervisorctl start scylla` and verify those extra processes and permissions are actually needed

Reply:
> I'm running it locally, and with hydra.
> and didn't need the `systctl --system` nor the `--privileged`
> 
> I need more details why this is needed, cause I see nor reason it should.
> in your log you called scylla directly without the correct commandline args.
> please use `supervisorctl start scylla` and verify those extra processes and permissions are actually needed

You have it working because your host system sysctl variables was tweeked, probably you have installed scylla on it at some point in the past. 
Docker does not support sysctl modifications from non-privileged containers.

Here is what I have:
[root@some-scylla /]# supervisorctl start scylla
scylla: ERROR (abnormal termination)
[root@some-scylla /]# supervisorctl
node-exporter                    RUNNING   pid 30, uptime 0:04:45
scylla                           FATAL     Exited too quickly (process log may have details)

[root@some-scylla /]# scylla --options-file /etc/scylla/scylla.yaml 
Scylla version 3.3.rc0-0.20200204.00b3f281995 with build-id 896009254492dd09d3b3094b5fc4d4bf559486e7 starting ...
command used: "/usr/bin/scylla --options-file /etc/scylla/scylla.yaml"
parsed command line options: [options-file: /etc/scylla/scylla.yaml]
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
ERROR 2020-02-06 17:54:20,155 [shard 3] seastar - Could not setup Async I/O: Resource temporarily unavailable. The most common cause is not enough request capacity in /proc/sys/fs/aio-max-nr. Try increasing that number or reducing the amount of logical CPUs available for your application

Reply:
I'm o.k. with documenting what we need todo with the host to run the docker image.
and then we do that once on where we need.

Reply:
> I'm o.k. with documenting what we need todo with the host to run the docker image.
> and then we do that once on where we need.

I'am sorry, did not get that. What exactly should I do ?

Reply:
I don't know exactly, since it was working for me out of the box. (And yes, I did install scylla a few time locally, maybe it changed something, maybe I did manually months ago for other reasons)

let start with a higher value in `/proc/sys/fs/aio-max-nr` and work it out from there, understanding what's really missing.

Reply:
> I don't know exactly, since it was working for me out of the box. (And yes, I did install scylla a few time locally, maybe it changed something, maybe I did manually months ago for other reasons)
> 
> let start with a higher value in `/proc/sys/fs/aio-max-nr` and work it out from there, understanding what's really missing.

Scylla installer already does everything for us, only thing we need to do is to run sysctl --system to pick it up from inside of container.